packages/text-link/package.json

```diff
@@ -13,13 +13,13 @@
   },
   "main": "build/index.js",
   "module": "build/index.es.js",
-  "style": "build/index.css",
   "files": [
     "build/*"
   ],
   "dependencies": {
-    "@hig/themes": "^0.3.0",
-    "classnames": "^2.2.5",
+    "@hig/behaviors": "^1.1.1",
+    "@hig/theme-context": "^2.0.0",
+    "@hig/theme-data": "^1.6.0",
```

Comment:
`theme-data` is at `2.0.1` now and do you need it as a dependency? I didn't see where it was being used.